.github/workflows/maven.yml

```diff
@@ -0,0 +1,75 @@
+name: Arquillian-Cube Maven Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'doc/**'
+      - '.circleci/**'
+env:
+  # failsafe.groups configuration depends on the env setup.
+  FAILSAFE_GROUPS: ''
+  DOCKER_HOST: 'unix:///var/run/docker.sock'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        java: ['11']
+
+    steps:
+      - name: Setup Minikube
+        uses: manusa/actions-setup-minikube@v2.9.0
+        with:
+          driver: docker
+          container runtime: containerd
+          minikube version: 'v1.30.1'
+          kubernetes version: 'v1.27.1'
```

Comment:
Just asking whether we'd want to use the latest K8s minor, or at least micro? See https://kubernetes.io/releases/

Or is there a reason for referencing `1.27.1`?

Reply:
Sure, we can use the latest version I guess. It was there from original PR. will update it to `1.30.0` which is the latest one I can see. also I will update the `actions-setup-minikube` to latest one to support it.